messages/env.compute.collaborator.add.md

```diff
@@ -0,0 +1,13 @@
+# summary
+
+Add a Heroku user as a collaborator on compute environments, allowing them to attach add-ons.
```

Comment:
I see that elsewhere in the PR we talk about a "Functions Account". It's a little confusing when half of the messages talk about that instead of compute environments and this description doesn't mention it. I think the easiest way to make it consistent is update this:

`Add a Heroku user as a collaborator on this Functions account, allowing them to attach Heroku add-ons to compute environments.`